src/kosha/app/ragam.clj

```diff
@@ -0,0 +1,26 @@
+(ns kosha.app.ragam
+  (:require [kosha.db.ragam :as db]
+            [kosha.app.util :as util]))
+
+(defn response-body
+  "Join ragam data, its kritis and its parent ragam into a single map."
+  [ragam kritis parent]
+  (-> {}
+      (assoc :ragam ragam)
+      (assoc :kritis kritis)
+      (assoc :parent-ragam parent)))
```

Comment:
I'd avoid this fn altogether (see comment below). But if I had to write it, I'd write a literal map:
```clojure
{:ragam ragam
 :kritis kritis
 :parent-ragam parent}
```